facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -66,6 +66,21 @@
 }
 
 
+@faciaCardExternal(card: ContentCard, withImage: Boolean, isBranded: Boolean) = {
+    @imgFromCard(card).filter(_ => withImage).fold {
+        @faciaCard(classesForCard(card), isBranded) {
+            @headline(card, isLast = true)
+        }
+    } { img =>
+        @faciaCard(classesForCard(card), isBranded) {
+            <tr valign="top">
+                @row(Seq("no-pad")){@img}
```

Comment:
`@row` contains a `<tr>` and `<td>` within it - any reason for wrapping inside another `<tr>`?

Reply:
Would something like this work? (There may be very good reasons why not!)

```scala
    @faciaCard(classesForCard(card, withImage), isBranded) {
        @if(withImage) {
            @row(Seq("no-pad")){@imgFromCard(card)}
        }
        @headline(card, isLarge = largeHeadline, isLast = false)
    }
```